parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -235,15 +235,15 @@ write_files:
 
 
 {{if not IsMariner}}
-{{if DisableUnattendedUpgrade }}
+{{if EnableUnattendedUpgrade }}
 - path: /etc/apt/apt.conf.d/99periodic
   permissions: "0644"
   owner: root
   content: |
-    APT::Periodic::Update-Package-Lists "0";
+    APT::Periodic::Update-Package-Lists "1";
     APT::Periodic::Download-Upgradeable-Packages "0";
```

Comment:
If we're not going to change autoclean or Download-Upgradeable-Packages why have them her at all.

Reply:
touche